packages/connect/src/index.ts

```diff
@@ -26,13 +26,16 @@ import { ConnectComponents } from './components.js'
 import { EntryNodes } from './base/entry.js'
 import { WebRTCUpgrader } from './webrtc/upgrader.js'
 import { UpnpManager } from './base/upnp.js'
+import { timeout } from '@hoprnet/hopr-utils'
 
 const DEBUG_PREFIX = 'hopr-connect'
 const log = Debug(DEBUG_PREFIX)
 const verbose = Debug(DEBUG_PREFIX.concat(':verbose'))
 const warn = Debug(DEBUG_PREFIX.concat(':warn'))
 const error = Debug(DEBUG_PREFIX.concat(':error'))
 
+const DEFAULT_CONNECTION_UPGRADE_TIMEOUT = 1000
```

Comment:
Is 1 second a reasonable value? Shouldn't be bigger?

Reply:
It is - at least in my opinion. The past has shown that protocol negotation is pretty quick, e.g. 100-200ms or takes forever